includes/admin/settings/class-wc-settings-shipping.php

```diff
@@ -229,10 +230,8 @@ protected function zone_methods_screen( $zone_id ) {
 			wp_die( esc_html__( 'Zone does not exist!', 'woocommerce' ) );
 		}
 
-		$allowed_countries = WC()->countries->get_allowed_countries();
-		$wc_shipping       = WC_Shipping::instance();
-		$shipping_methods  = $wc_shipping->get_shipping_methods();
-		$continents        = WC()->countries->get_continents();
+		$allowed_countries    = WC()->countries->get_allowed_countries();
```

Comment:
This should probably be changed to allowed shipping countries, otherwise it populates all countries for the enabled continents.

Reply:
Good point - looks like you got this in #22339, but I've fixed it in https://github.com/woocommerce/woocommerce/pull/22131/commits/45e7f344b1743afb6e79bdcc05b1ba7dc4246849 also. Thanks!